File: comparePython.js
//double slash to make comments

//set a variable
var y="bananas";
var num=6;
var arr =["John","Callie",y];
var mybool = true;

//output to the console
console.log(y); //output will be "bananas"

//output some strings
console.log("My name is Amanda.")
console.log("Hi Vinh")

var x = 10
function myFunction(){
    x=5;
    return;
}
myFunction();
console.log(x)

var myvar=10;
if(myvar<5){
    console.log("greater than 5");
}
else{
    console.log("less than 5");
}

function sayHello(name){
    var hellostr = "Hello, "+name;
    return hellostr;
}
console.log(sayHello("Mike"));

var x =sayHello("Vinh");
console.log(x)

sayHello();
console.log("You")

function myLoop(){
    for(var i=0; i<=10;i++){
        console.log(i);
    }
    return;
}
myLoop();

var bestArray=["Dog",1,2,3,"Vinh"];

function Loopy(arr){
    var newArray=[];
    for(var i= arr.length-1; i>=0; i--){
        newArray.push(arr[i]);
    }
    return newArray;
}
console.log(Loopy(bestArray));

arr.push("I love coding");

File: comparePython.py
# hash sign to make comments


y="bananas"
num=6
arr=["John","Callie",y] #arrays are called lists in python
mybool= True

#output to consolep
print(y)

print("My name is Amanda.")
name="Amanda"
favnum= 100

print("My name is", name,"and my favorite number is", favnum)
print("My name is " +name+ "and my favorite number is " +str(favnum)) 


list_words = ["Hello","yes", "bye", "last"]
to_print = ""

for i in range(len(list_words)):
    to_print +=list_words[i]
    if i != len(list_words) - 1:
        to_print += ",    "

print(to_print)

list_words = ["Hello","yes", "bye", "last"]
DELIMETER = ","
print(DELIMETER.join(list_words))




    